internal/job/repository.go

```diff
@@ -51,3 +52,41 @@ func (repo *jobRepository) GetJobs(ctx context.Context, pageNumber int, itemsPer
 
 	//todo: Offset and Limit is not working
 }
+
+func (repo *jobRepository) GetFavouriteJobs(ctx context.Context, personID string) (models.JobSlice, error) {
+	uuid, err := uuid.FromString(personID)
+
+	if err != nil {
+		return nil, err
+	}
+
+	jobFavs, err := models.JobSeekerFavs(
+		models.JobSeekerFavWhere.PersonID.EQ(uuid.String())).All(ctx, repo.executor)
+
+	if err != nil {
+		if err.Error() == errSqlNoRows {
+			return nil, err
+		}
+		return nil, err
+	}
+
+	var jobs models.JobSlice
+	for _, favoriteJob := range jobFavs {
```

Comment:
can be simplified with 
`SELECT j.* FROM job j INNER JOIN job_seeker_fav jsf ON j.ID = jsf.job_id WHERE jsf.person_id = '<uuid.String()>'`

We can push this change in, I will update the query.

Reply:
I have done some changes, please merge locally